src/services/documentation.service.ts

```diff
@@ -77,6 +77,31 @@ class DocumentationService {
     'ngInject';
   }
 
+  buildPageList = (pagesToFilter: any[], withRootFolder?: boolean, folderSituation?: FolderSituation) => {
+    const pageList = _.filter(
```

Comment:
We can use the default filter function instead of the lodash one:  `pagesToFilter.filter()`